wizeline/fox-poc-backend
# Make status field required as well

### What does this PR do?
Make the `status` field required for a `POST` operation.

### Where should the reviewer start?
At `src/validators/schemas/CreatePost.js`.

### How can this be manually tested?
Try to create a post without the `status` field and make sure the response is a `bad-request`
